lib/common/commonUtil.py

```diff
@@ -262,6 +262,15 @@ class CommonUtil(object):
     logger = get_logger(__file__)
 
     @staticmethod
+    def execute_runipy_cmd(input_template_fp, output_ipynb_fp, **kwargs):
```

Comment:
For executing `runipy` (and the code inside the ipynb file), we have to move the dependencies from `tools\requirements.txt` to `requirements.txt`.

Ref: https://github.com/Mozilla-TWQA/Hasal/blob/master/tools/requirements.txt#L4-L6